bridge/client/app/_models/trace.ts

```diff
@@ -170,6 +170,15 @@ class Trace {
     return this.data.service;
   }
 
+  getChartLabel(): string {
+    let label;
```

Comment:
[minor] Could be written a bit shorter with Typescript 3.7
```suggestion
    return this.data.labels?.get("buildId") ?? this.time;
```

Reply:
It might happen, that there are labels defined, but without `buildId`. Is this also working in that case?

Reply:
If the result on the left side is `null` or `undefined`, it will fall back to `this.time`. If you additionally want to check for empty strings (falsy values) you can use `||` instead of `??`.

Reply:
Sounds greate, thx.